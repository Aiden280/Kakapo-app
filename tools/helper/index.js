import fs from 'fs-extra';

// Upload stats.json to `http://webpack.github.io/analyse` for analysis.
// Also you can analyze size with `https://github.com/robertknight/webpack-bundle-size-analyzer`:
// `cat stats.json | analyze-bundle-size`
export const writeStats = stats =>
  fs.writeFile('stats.json', JSON.stringify(stats.toJson(), null, 4));

export default {};
