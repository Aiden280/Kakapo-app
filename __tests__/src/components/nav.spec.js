import React from 'react';
import { shallow } from 'enzyme';
import { getData } from '../helper';
import { Nav } from 'components/';

function setup(props = {}) {
  const propData = {
    ...getData('themes'),
    ...getData('intl'),
    ...props
  };
  return { props, wrapper: shallow(<Nav {...propData} />) };
}

test('<Nav/> render', () => {
  expect.assertions(1);
  const { wrapper } = setup();
  expect(wrapper.prop('className')).toBe('topbar');
});
