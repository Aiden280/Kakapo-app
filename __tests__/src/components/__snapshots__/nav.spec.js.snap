// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Nav/> render 1`] = `
<div
  className="topbar"
  style={
    Object {
      "backgroundColor": "hsl(261.6, 51.9%, 37.8%)",
    }
  }
>
  <a
    className="download-app"
    href="http://www.kakapo.co/app.html"
    rel="noopener noreferrer"
    target="_blank"
  >
    Download Kakapo for iOS / Android, Desktop & Chrome!
  </a>
  <div
    className="share"
  >
    <div
      className="fb-share-button"
      data-href="http://www.kakapo.co"
      data-layout="button_count"
    />
    <a
      className="twitter-share-button"
      data-url="http://www.kakapo.co"
      href="https://twitter.com/share"
    >
      Tweet
    </a>
  </div>
</div>
`;
